clash-lib/src/Clash/Unique.hs

```diff
@@ -97,15 +99,15 @@ instance Uniquable Int where
 newtype UniqMap a = UniqMap (IntMap a)
   deriving (Functor, Foldable, Traversable, Semigroup, Monoid, NFData, Binary)
 
-instance Pretty a => Pretty (UniqMap a) where
-  pretty (UniqMap env) =
+instance Pretty' a => Pretty' (UniqMap a) where
+  pr (UniqMap env) =
```

Comment:
Isn't it better to call this function `pretty'`? Seems to match the class rename better. Maybe even call it `ClashPretty`/`clashPretty` or something similar to clarify that this is _our_ implementation.